src/application.ts

```diff
@@ -107,7 +111,8 @@ export default class Application implements Owner {
     if (!mainTemplate) { throw new Error("Could not find main template."); }
 
     let mainLayout = templateFactory(mainTemplate).create(this.env);
```

Comment:
@chancancode Hmm, how do we hard-code that when the compiler is not available at runtime?